docs/user_docs/installation/upgrade/upgrade-kubeblocks-to-0.8.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Upgrade to KubeBlocks v0.8
+description: Upgrade to KubeBlocks v0.8, operation, tips and notes
+keywords: [upgrade, 0.8]
+sidebar_position: 1
+sidebar_label: Upgrade to KubeBlocks v0.8
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Upgrade to KubeBlocks v0.8
+
+In this tutorial, you will learn how to upgrade to KubeBlocks v0.8.
+
+
+:::note
+
+Execute `kbcli version` to check the current KubeBlocks version you are using, and then upgrade.
+
+
+:::
+
+## Upgrade from KubeBlocks v0.6
+
+If you are currently using KubeBlocks v0.6, please upgrade to v0.7.2 first.
+
+
+1. Download kbcli v0.7.2.
+
+    ```shell
+    curl -fsSL https://kubeblocks.io/installer/install_cli.sh | bash -s 0.7.2
+    ```
+
+2. Upgrade to KubeBlocks v0.7.2.
+
+    ```shell
+    kbcli kb upgrade --version 0.7.2
+    ```
+
+## Upgrade from KubeBlocks v0.7
+
+<Tabs>
+
+<TabItem value="Helm" label="Helm" default>
+
+1. Set keepAddons.
+
+    KubeBlocks v0.8 streamlines the default installed engines and removes greptime, influxdb, neon, oracle-mysql, orioledb, tdengine, mariadb, nebula, risingwave, starrocks, tidb, and zookeeper. To avoid deleting addon resources that are already in use during the upgrade, execute the following commands:
```

Comment:
“removes greptime, influxdb, neon, oracle-mysql, orioledb, tdengine, mariadb, nebula, risingwave, starrocks, tidb, and zookeeper.”  is a bit scary, "separates the addons from KubeBlocks operator to KubeBlocks-Addons repo, such as ..." seems better